types/utils.go

```diff
@@ -154,3 +155,45 @@ func ContainsString(s []string, e string) bool {
 	}
 	return false
 }
+
+func CompareVersionStrings(verStr1, verStr2 string) (int, error) {
```

Comment:
Please add a comment on what this does and let the reader know to look at the tests for reference